.github/workflows/pr.yml

```diff
@@ -46,10 +46,10 @@ jobs:
           java-version: 17
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v3
-      - name: "Validate Gradle wrapper"
+      - name: "ktLint Check"
```

Comment:
That's still the wrapper check, or? I'm seeing this renaming, but I think it shouldn't be here.

Reply:
Ahh, good catch. Fixing it!